### ISSUE TITLE
Document how to use buckets

### ISSUE THREAD

**Issue by nirvdrum** [kvandake/lexorank-js#26]
While looking at the source, I noticed that `LexoRank.max()` returns the max value for the first bucket and not the max value permissible. I thought maybe this was a bug because the library sets up three buckets. However, the library also can't call `LexoRank.between()` with two `LexoRank` instances corresponding to different buckets.

I was under the impression buckets were used to expand the range of possible values or possibly to shard values or help out database indices. But, it looks like the only way to get out of `BUCKET_0` is to manually create the `LexoRank` with a reference to another `LexoRankBucket`. Can you please clarify the role buckets play and how they should be used?

**Comment by bhargavgohil25** [comment]
Yeah. It would be really helpful to include lexorank bucket documentation.

**Comment by RishiSargar** [comment]
Yes, It would really helpful get more information on how to apply rebalancing with the help of buckets

**Comment by episage** [comment]
Buckets are solution to lack of atomic transactions in NoSQL system. You keep using bucket 0 and recreate/rebalance in 1, then switch to using 1. All operations on 0 must be paused while doing it.
